core/domain/suggestion_registry.py

```diff
@@ -1095,3 +1095,95 @@ def reject(self, reviewer_id, rejection_message):
         SuggestionTranslateContent),
     suggestion_models.SUGGESTION_TYPE_ADD_QUESTION: SuggestionAddQuestion
 }
+
+
+class ReviewerAndSuggestionCounts(python_utils.OBJECT):
+    """Domain object for the ReviewerAndSuggestionCountsModel.
+
+    Attributes:
+        translation_reviewer_counts_per_lang: dict. A dictionary that contains
+            the total number of translation reviewers for each language that
+            translation suggestions are offered in.
+        translation_suggestion_counts_per_lang: dict. A dictionary that
+            contains the total total number of translation suggestions that are
+            currently in review per language.
+        question_reviewer_count: int. The total number of reviewers who have
+            permission to review question suggestions.
+        question_suggestion_count: int. The total number of question
+            suggestions that are currently in review.
+    """
+
+    def __init__(
+            self, translation_reviewer_counts_per_lang,
+            translation_suggestion_counts_per_lang,
+            question_reviewer_count, question_suggestion_count):
+        self.translation_reviewer_counts_per_lang = (
+            translation_reviewer_counts_per_lang
+        )
+        self.translation_suggestion_counts_per_lang = (
+            translation_suggestion_counts_per_lang
+        )
+        self.question_reviewer_count = question_reviewer_count
+        self.question_suggestion_count = question_suggestion_count
+
+    def validate(self):
+        """Validates the ReviewerAndSuggestionCounts object.
+
+        Raises:
+            ValidationError. One or more attributes of the
+                ReviewerAndSuggestionCounts object is invalid.
+        """
+        for language_code, reviewer_count in (
+                self.translation_reviewer_counts_per_lang.items()):
+            if reviewer_count < 0:
+                raise utils.ValidationError(
+                    'Expected the translation reviewer count to be positive, '
```

Comment:
positive --> non-negative, here and below.

Reply:
Done